### ISSUE TITLE
make www redirect work with subdirs

### ISSUE THREAD

**Issue by jberkus** [projectatomic/atomic-site#287]
The way it is now:

If I type a subdir without the "www", I get redirected to the projectatomic home page:

   http://projectatomic.io/docs

The way it should be:

I should get redirected to the page indicated.

This isn't a big deal, but then it might not be that hard to fix, either.

**Comment by mscherer** [comment]
yeah, that's in IT hands, not much we can do right now. The move to openshift v3 will not likely fix that.
